README.md

```diff
@@ -99,6 +99,7 @@ People
 * [Fuyuzhen Peng](people/fuyuzhen-peng.md) - fp73 - [http://cs5356.s3-website-us-east-1.amazonaws.com/grading.html](http://cs5356.s3-website-us-east-1.amazonaws.com)
 * [Ryan Ramphal](people/ryan-ramphal.md) - rdr233 - [http://s3site.ryanramphal.com.s3-website-us-east-1.amazonaws.com/grading.html](http://s3site.ryanramphal.com.s3-website-us-east-1.amazonaws.com/grading.html)
 * [Hathaitorn Rojnirun](people/hathaitorn-rojnirun.md) - hr346 - [https://s3.amazonaws.com/hr346/grading.html](https://s3.amazonaws.com/hr346/grading.html)
+* [Vicente Rotman Hinzpeter](people/vicente-rotman-hinzpeter.md) - vr328 - [https://s3.us-east-2.amazonaws.com/startupsystkmsvicente/grading.html](https://s3.us-east-2.amazonaws.com/startupsystkmsvicente/grading.html)
```

Comment:
Where is `people/vicente-rotman-hinzpeter.md`?

Reply:
thats the only thing that left?
I'll do it now, sorry